bloom-housing/ui-components
# fix: remove cloudinary references

# Pull Request Template

#98

## Description

Blocked by https://github.com/bloom-housing/bloom/pull/3393

## How Can This Be Tested/Reviewed?

The Dropzone story had to change to support this, but I think it's an improvement. The story included business log about the Dropzone using the Cloudinary API. You can check out the [Dropzone stories](https://deploy-preview-97--storybook-bloom-dev.netlify.app/?path=/story/forms-dropzone--default-dropzone).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Either review the Storybook preview or pull the changes down locally and test that the acceptance criteria is met
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

On merge, squash commits.
